docs/technical_reference/flowsheets/gac.rst

```diff
@@ -0,0 +1,91 @@
+Granular Activated Carbon
+=========================
+
+Introduction
+------------
+
+The simple granular activated carbon (GAC) flowsheet can be simulated to predict the performance of a GAC system to treat residual organics. This flowsheet can
+be useful to expedite the set-up, usage, and costing of a GAC system for conventional water treatment applications using the constant pattern homogeneous
+surface diffusion model (CPHSDM) model.
+
+Implementation
+--------------
+
```

Comment:
Expand this section to include a link to the property model documentation and WaterTAP costing package. Also add a brief sentence (feel free to copy from other flowsheet examples) on the levelized cost of water and the equation for it.

TLDR: Make this section more consistent with the other flowsheet documentation.

Reply:
Actually, it looks like not all of the examples link to the costing package and/or describe the LCOW equation, so you don't necessarily need to add these. But you should link to the MCAS property package